stan/math/prim/functor/hcubature.hpp

```diff
@@ -380,148 +350,158 @@ std::tuple<double, double, int> integrate_GenzMalik(
   return std::make_tuple(I, E, kdivide);
 }
 
+/**
+ * Compute the integral of the function to be integrated (integrand) from a to b
+ * for more than one dimensions.
+ *
+ * @tparam T_a Type of return_type_t 1
+ * @tparam T_b Type of return_type_t 2
+ * @tparam Vec1 Type of vector 1
+ * @tparam Vec2 Type of vector 2
```

Comment:
Docs for these two template parameters would go into the docs for the constructor of `Box()`